### PR TITLE
GUVNOR-3052: [GDT Column Wizard] Field Binding is disabled for LimitedEntry table

### PR DESCRIPTION
**Pull request by karreiro**
See https://issues.jboss.org/browse/GUVNOR-3052

aaaa

### CODE REVIEW

**Comment by karreiro** [comment]
@manstis @jomarko Could you please take a look?

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/plugins/ConditionColumnPlugin.java [hunk 1]
```diff
@@ -199,7 +199,7 @@ void prepareValues() {
             editingCol().setOperator(null);
         }
 
-        if (constraintValue() != BaseSingleFieldConstraint.TYPE_LITERAL) {
+        if (!(tableFormat() == GuidedDecisionTable52.TableFormat.LIMITED_ENTRY || constraintValue() == BaseSingleFieldConstraint.TYPE_LITERAL)) {
```
Comment (jomarko) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/plugins/ConditionColumnPlugin.java]: Hi @karreiro , just question, this condition is quite complex and it seems to me that it should be kept in sync with `valueOptionsPage.canSetupBinding()`. I see that `canSetupBinding()` and this `if` are sligtly different, however, what do you think , could this `if` looks like `if (`valueOptionsPage.canSetupBinding())`?

Comment (karreiro) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/plugins/ConditionColumnPlugin.java]: Thank you for the idea, @jomarko. I did something a little bit different, but I kept the idea of DRYing the code ;-)

**Comment by manstis** [comment]
@karreiro Just the conflict to resolve (we guessed we'd have one somewhere)

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by karreiro** [comment]
****

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Pull request merged**